src/shipit_static_analysis/shipit_static_analysis/cli.py

```diff
@@ -11,11 +11,12 @@
 from shipit_static_analysis.report import get_reporters
 from shipit_static_analysis.lock import LockDir
 from shipit_static_analysis import config
+from shipit_static_analysis import stats
```

Comment:
Nit: move this to previous line `from shipit_static_analysis import config, stats`